bundles/org.openhab.binding.systeminfo/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -159,271 +159,271 @@
 	<channel-type id="path">
 		<item-type>String</item-type>
 		<label>Path</label>
-		<description>The full path</description>
+		<description>The full path.</description>
 		<state readOnly="true" pattern="%s"/>
 		<config-description-ref uri="channel-type:systeminfo:lowpriority"/>
 	</channel-type>
 
 	<channel-type id="path_process">
 		<item-type>String</item-type>
 		<label>Path</label>
-		<description>The full path</description>
+		<description>The full path.</description>
 		<state readOnly="true" pattern="%s"/>
 		<config-description-ref uri="channel-type:systeminfo:lowpriority_process"/>
 	</channel-type>
 
 	<channel-type id="available">
 		<item-type>Number</item-type>
 		<label>Available</label>
-		<description>Available size in MB</description>
+		<description>Available size.</description>
```

Comment:
While your channel is expected to be linked to a Number item without unit, it is better to mention the unit in the description. 

Same remark for all other similar channels. 

Please also adjust the default translations file.

Reply:
I will adjust the default translations.

However, what about linking the plain number channel to a Number:DataAmount item. I understood it should then convert from the provided MB, as this comes with the default state description. That’s why I removed the units from the description.
It would probably even be better to convert all of these channels to QuantityType channels. Attaching an plain Number type to them would still give the same result as today.

Reply:
> However, what about linking the plain number channel to a Number:DataAmount item. I understood it should then convert from the provided MB, as this comes with the default state description. That’s why I removed the units from the description.

No, if you defined a plain number channel, there will be no unit conversion. If "MB" is in the state pattern, it will be just added when formatting the state.
If you want UOM with conversion, you need to define a channel with a number with dimension.

> It would probably even be better to convert all of these channels to QuantityType channels. Attaching an plain Number type to them would still give the same result as today.

Yes, it is of course better to use UoM to get unit conversion feature. I have just a doubt if it would be a breaking change or not for users who defined things with UI.

Reply:
if you defined a plain number channel, we can expect the binding is building a DecimalType state and so no unit involved.